packages/dx-react-scheduler/src/plugins/appointment-form.tsx

```diff
@@ -215,14 +250,15 @@ class AppointmentFormBase extends React.PureComponent<AppointmentFormProps, Appo
       readOnly,
       messages,
     } = this.props;
-    const { previousRule, visible, appointmentData } = this.state;
+    const { visible, appointmentData, previousAppointment } = this.state;
     const getMessage = this.getMessage(defaultMessages, messages);
-
     return (
       <Plugin
         name="AppointmentForm"
         dependencies={pluginDependencies}
       >
+        <Action name="toggleAppointmentFormVisibility" action={this.toggleVisibility} />
```

Comment:
If you are creating a constant with this name `toggleAppointmentFormVisibility`? Why didn't use it here?